app/views/layouts/application.html.erb

```diff
@@ -2,57 +2,79 @@
-       <%= link_to('Logout', destroy_user_session_path, method: :delete) %>
-       <% else %>
-       <%= link_to('Login', new_user_session_path) %>
-       <% end %>
-       </li>
-      </ul>
-        
-        <%= form_with(url: products_path, method: 'get', local: true) do |form| %>
-         <%= form.label :q, 'Search for:' %>
-         <%= form.text_field :q, id: 'q', value: params[:q] %>
-         <%= form.submit 'Search' %>
-        <% end %>
-        
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <%= link_to "Red", root_path, class: "navbar-brand" %>
+        </div>
+
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+          <form class="navbar-form navbar-left">
+            <div class="form-group">
+              <%= form_with(url: products_path, method: 'get', local: true) do |form| %>
+                <%= form.text_field :q, id: 'q', value: params[:q], class: "form-control" %>
+                <%= form.submit 'Search', class:"btn btn-default" %>
+              <% end %>
+            </div>
+          </form>
+          <ul class="nav navbar-nav navbar-right">
+            <li><%= link_to "Contact",  simple_pages_contact_path %></li>
+            <li><%= link_to "About", simple_pages_about_path %></li>
+            <li><%= link_to "Products", products_path %></li>
+            <li>
+              <% if user_signed_in? %>
+                <%= link_to('Logout', destroy_user_session_path, method: :delete) %>
+              <% else %>
+                <%= link_to('Login', new_user_session_path) %>
+              <% end %>
+            </li>
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
     </nav>
-      
-    
-      
-    <% if notice %>
-    <p id="notice" class="alert alert-info"><%= notice %></p>
+
+    <% if flash[:notice] %>
+      <p id="notice" class="alert alert-info"><%= flash[:notice] %></p>
+    <% end %>
+    <% if flash[:success] %>
+      <p id="notice" class="alert alert-success"><%= flash[:success] %></p>
     <% end %>
-    <p id="alert" class="alert alert-info"><%= alert %></p>  
-      
-    <%= yield %>
-
-      <footer class="site-footer">
-      
-      &copy; 2018 Code Red Tech
-      
-      </footer>
-        
+    <% if flash[:error] %>
+      <p id="notice" class="alert alert-warning"><%= flash[:error] %></p>
+    <% end %>
```

Comment:
flash[:error], flash[:success], flash[:notice]. red, green, blue. These little notices whenever you've done something bad, good or just need to declare basic info in rails.